docs/src/pages/components/modal/modal.md

```diff
@@ -104,6 +123,6 @@ Additionally, you may give a description of your modal with the `aria-describedb
 ## Server-side modal
 
 React [doesn't support](https://github.com/facebook/react/issues/13097) the [`createPortal()`](https://reactjs.org/docs/portals.html) API on the server.
-In order to make it work, you need to disable this feature with the `disablePortal` prop:
+In order to display see the modal, you need to disable the portal feature with the `disablePortal` prop:
```

Comment:
'display' or 'see'?

Reply:
I think that see is better in this context.